### PR TITLE
SONAR-7917 Drop project dashboards - Backend

### PR DESCRIPTION
**Pull request by teryk**

### CODE REVIEW

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [v61](https://github.com/SonarSource/sonarqube/blob/ccb5a69219a6d84745b65cecedf2b07ff18cb082/sonar-db/src/main/java/org/sonar/db/version/v61): Add a 'package-info.java' file to document the 'v61' package [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1228)

File: server/sonar-web/src/main/webapp/WEB-INF/app/controllers/dashboard_controller.rb [hunk 1]
```diff
@@ -120,7 +120,7 @@ def add_widget
                                             :name => definition.getTitle(),
                                             :column_index => 1,
                                             :row_index => 1,
-                                            :configured => !(definition.hasRequiredProperties() || (dashboard.global && !definition.isGlobal)))
+                                            :configured => !(definition.hasRequiredProperties() || (!definition.isGlobal)))
```
Comment (simonbrandhof) [inline server/sonar-web/src/main/webapp/WEB-INF/app/controllers/dashboard_controller.rb]: you don't need to keep `!definition.isGlobal`

File: server/sonar-web/src/main/webapp/WEB-INF/app/helpers/dashboard_helper.rb [hunk 2]
```diff
@@ -155,7 +155,7 @@ def restore_global_resource
   end
 
   def widget_title(widget)
-    resource_name=link_to(h(@resource.name), {:controller => 'dashboard', :action => 'index', :id => @resource.id}) if @resource && @dashboard.global && !widget.java_definition.global
+    resource_name=link_to(h(@resource.name), {:controller => 'dashboard', :action => 'index', :id => @resource.id}) if @resource && !widget.java_definition.global
```
Comment (simonbrandhof) [inline server/sonar-web/src/main/webapp/WEB-INF/app/helpers/dashboard_helper.rb]: why `!widget.java_definition.global` ? It is always false no ?

File: server/sonar-web/src/main/webapp/WEB-INF/app/models/dashboard.rb [hunk 3]
```diff
@@ -55,16 +54,8 @@ def shared?
...
 
-  def global=(global)
-    write_attribute(:is_global, global)
-  end
-
-  def global
-    read_attribute(:is_global)
-  end
-
   def global?
```
Comment (simonbrandhof) [inline server/sonar-web/src/main/webapp/WEB-INF/app/models/dashboard.rb]: to be fully backward-compatible, you can also keep 
```
def global
  true
end
```

No need for keeping `global=`

File: server/sonar-web/src/main/webapp/WEB-INF/app/views/dashboard/_configure_widget.html.erb [hunk 4]
```diff
@@ -6,7 +6,7 @@
 
 <div class="widget-header">
   <div class="widget-actions">
-    <% if widget.java_definition.isEditable() || (!widget.java_definition.global && @dashboard.global) %>
+    <% if widget.java_definition.isEditable() || (!widget.java_definition.global) %>
```
Comment (simonbrandhof) [inline server/sonar-web/src/main/webapp/WEB-INF/app/views/dashboard/_configure_widget.html.erb]: widget.java_definition.global ?

File: server/sonar-web/src/main/webapp/WEB-INF/app/views/dashboard/_widget_properties.html.erb [hunk 5]
```diff
@@ -3,7 +3,7 @@
   <table class="table width100">
     <tbody>
 
-    <% if !widget.java_definition.global && @dashboard.global %>
+    <% if !widget.java_definition.global %>
```
Comment (simonbrandhof) [inline server/sonar-web/src/main/webapp/WEB-INF/app/views/dashboard/_widget_properties.html.erb]: widget.java_definition.global ?

File: server/sonar-web/src/main/webapp/WEB-INF/app/models/dashboard.rb (hunk 3 above)
Comment (teryk) [inline server/sonar-web/src/main/webapp/WEB-INF/app/models/dashboard.rb]: ok

**Comment by simonbrandhof** [comment]
LGTM

**Pull request merged**